src/pages/en/guides/publish-to-npm.md

```diff
@@ -279,4 +279,4 @@ In addition to the required `astro-component` keyword, special keywords are also
 
 ## Share
 
-We encourage you to share your work, and we really do love seeing what our talented Astronauts create. Come and share what you create with us in our [Discord](https://discord.gg/YQRVveAgED) or mention [@astrodotbuild](https://twitter.com/astrodotbuild) in a Tweet!
+We encourage you to share your work, and we really do love seeing what our talented Astronauts create. Come and share what you create with us in our [Discord](https://astro.build/chat/) or mention [@astrodotbuild](https://twitter.com/astrodotbuild) in a Tweet!
```

Comment:
And changed this discord invitation to the astro.build link

Reply:
I was wandering why it was a custom link! it's fine if it goes to the docs chat, it's even better! (could i take the liberty to footnote this link so we know that i point directly to the docs channel ? (and so it's only visible on the soucecode))